### ISSUE TITLE
Extend ils -l to show system metadata about collections

### ISSUE THREAD

**Issue by tedgin** [irods/irods#7790]
One of our users has requested that the `ils -l` be extended show system metadata about collections. Maybe `COLL_OWNER_NAME`, `COLL_OWNER_ZONE`, and `COLL_MODIFY_TIME` values could be added to the output and displayed in a way that is similar to the corresponding data object values are displayed.

**Comment by trel** [comment]
We have been hesitant to edit the output / layout of `ils` for a long time since many scripts originally parsed the output.

To avoid that being a problem... we could introduce a new option to print more collection information - perhaps... `-C`?

Can you propose a possible output format for us to react to?

**Comment by tedgin** [comment]
Here's a way it could work. Following the form of `ils -A`, add two indented lines immediately below the collection path in the output. One line would show the owner name and zone, and the second would show the time of last modification. Here's the form:

```
/path/to/collection:
        Owner - owner-name#owner-zone
        Modified - modify.time
  data-object-1
  ⋮
  data-object-n
  C- /path/to/collection/member-collection-1
  ⋮
  C- /path/to/collection/member-collection-m
```

Here's an example:

```console
prompt> ils -C
/iplant/home/tedgin:
        Owner - tedgin#iplant
        Modified - 2024-04-11.14:23
  RFHS-deployment.gan
  C- /iplant/home/tedgin/analyses
```

Here's an an example when combined with the `-A`, and `-L` options:

```console
prompt> ils -ALC
/iplant/home/tedgin:
        Owner - tedgin#iplant
        Modified - 2024-04-11.14:23
        ACL - anonymous#iplant:read object   bisque#iplant:read object   tedgin#iplant:modify object
        Inheritance - Disabled
  tedgin            0 CyVerseRes;hugoBroker;hugo        13891 2023-11-20.11:32 & RFHS-deployment.gan
    15af146b4ca714ed8cde683ebd79b26b    generic    /irods_vault/home/tedgin/RFHS-deployment.gan
        ACL - g:rodsadmin#iplant:own   tedgin#iplant:own   
  tedgin            1 taccRes;corral4        13891 2023-11-20.11:32 & RFHS-deployment.gan
    15af146b4ca714ed8cde683ebd79b26b    generic    /corral/irods/iplant/Vault/home/tedgin/RFHS-deployment.gan
        ACL - g:rodsadmin#iplant:own   tedgin#iplant:own
  C- /iplant/home/tedgin/analyses
```

**Comment by tedgin** [comment]
I'm not certain what this user use's the owner for.  It's nice to know who the owner of collection or data object is, because this account can always modify its permission level on that collection or data object even if the account has no permission on it.

**Comment by trel** [comment]
I think that is no longer true... but we can compare notes.

**Comment by korydraughn** [comment]
> ... because this account can always modify its permission level on that collection or data object even if the account has no permission on it.

As @trel said, that's no longer the case in 4.3. If the permissions for the user who created the collection are removed, they will not be able to restore them. Only a rodsadmin can restore them.

**Comment by trel** [comment]
4.3.1 ... https://github.com/irods/irods/issues/6579

**Comment by tedgin** [comment]
Now that owner has no purpose other than to identify the user account that originally created the collection or data object, I see no reason to display a collection owner other than to be consistent with `ils -l` showing a data object's owner.  The term "owner" is now a misnomer. Maybe "creator" would be a better term to describe it. If you decide to display the collection owner as part of an `ils -C` output, maybe it should be labeled `Creator` to be less confusing to users. 

```
/path/to/collection:
        Creator - owner-name#owner-zone
        Modified - modify.time
  data-object-1
  ⋮
  data-object-n
  C- /path/to/collection/member-collection-1
  ⋮
  C- /path/to/collection/member-collection-m
```

**Comment by trel** [comment]
Agreed that owner doesn't help much... I think we would just add time at this point.
